plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeBasic.java

```diff
@@ -117,6 +119,83 @@ public void testNoColumnStats()
         assertQuery("SELECT c_str FROM no_column_stats WHERE c_int = 42", "VALUES 'foo'");
     }
 
+    /**
+     * @see databricks.timestamp_ntz
+     */
+    @Test
+    public void testTimestampNtz()
+    {
+        // TODO Move this test to product test once new Databricks LTS or OSS Delta Lake supports timestamp_ntz type
```

Comment:
Upcoming OSS Delta Lake 2.4.0 supports this type. 
https://github.com/delta-io/delta/releases/tag/v2.4.0rc1